README.md

```diff
@@ -35,10 +39,16 @@ As with any cli tool just run the following to see available actions/arguments.
 
 ## Repository Report
 
-Run the following command generate a CSV report with respository settings and branch protection rules.
+Run the following command generate a CSV or JSON report with respository settings and branch protection rules.
```

Comment:
should this be 'Run the following command `to` generate a ...'
To make it a bit clearer? on `line 48` too, Very picky sorry :)

Reply:
oh yes good spot!!!  Eagle eyes!!! Nice one